File: Basic/Java/src/samples/QiViewProperty.java
package samples;


public class QiViewProperty {
    private String SourceId;
    private String TargetId;
    private QiView QiView;

    public String getSourceId() {
        return SourceId;
    }

    public void setSourceId(String SourceId) {
        this.SourceId = SourceId;
    }

    public String getTargetId() {
        return TargetId;
    }

    public void setTargetId(String targetId) {
        this.TargetId = targetId;
    }

    public QiView getQiView() {
        return QiView;
    }

    public void setQiView(QiView qiView) {
        this.QiView = qiView;
    }
}

File: Basic/DotNet/QiRestApiCore/QiViewProperty.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QiRestApiCore
{
    public class QiViewProperty
    {
        public string SourceId
        {
            get;
            set;
        }

        public string TargetId
        {
            get;
            set;
        }

        public QiView QiView
        {
            get;
            set;
        }
    }
}
